BundesIdent/TCA/Coordinator.swift

```diff
@@ -101,9 +107,14 @@ struct Coordinator: ReducerProtocol {
         case didEnterBackground
         case routeAction(Int, action: Screen.Action)
         case updateRoutes([Route<Screen.State>])
+        case remoteConfiguration(RemoteConfiguration.Action)
     }
     
     var body: some ReducerProtocol<State, Action> {
+        Scope(state: \.remoteConfiguration, action: /Action.remoteConfiguration) {
+            RemoteConfiguration().dependency(\.mainQueue, mainQueue)
```

Comment:
Why do we have to pass the `mainQueue` here?

Reply:
Ah, good catch! It’s a leftover from figuring out the tests :-)